libs/cardano-ledger-core/cddl/Cardano/Ledger/HasCDDL.hs

```diff
@@ -0,0 +1,41 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE UndecidableSuperClasses #-}
+
+module Cardano.Ledger.HasCDDL (
```

Comment:
Sorry, for suggesting to change this again, but considering my other suggestions, I think it will be a better name
```suggestion
module Cardano.Ledger.Huddle (
```